test/rulesets/LinearAlgebra/factorization.jl

```diff
@@ -23,6 +23,25 @@ end
 const LU_ROW_MAXIMUM = VERSION >= v"1.7.0-DEV.1188" ? RowMaximum() : Val(true)
 const LU_NO_PIVOT = VERSION >= v"1.7.0-DEV.1188" ? NoPivot() : Val(false)
 
+# well-conditioned random n×n matrix with elements of type `T` for testing `eigen`
+function rand_eigen(T::Type, n::Int)
+    # uniform distribution over `(-1, 1)` / `(-1, 1)^2`
+    _rand(T, n...) = rand(T, n...) .* rand(T <: Complex ? [1, im, -1, -im] : [1, -1], n...)
+
+    # make sure that each eigenvector has one clearly defined entry with maximum magnitude
+    # so that the normalization of EVs is well defined
```

Comment:
Technically it's so that the phase is well defined.

Reply:
Ah, you are right! I messed up my terminology here.

Reply:
```suggestion
    # so that the complex phase of EVs is well defined
```